ERROR:  While executing gem ... (Gem::InvalidSpecificationException)
	duplicate dependency on rdoc (>= 0, development), (>= 0) use:
	add_runtime_dependency 'rdoc', '>= 0', '>= 0'

--- origsrc/tilt-1.4.1/tilt.gemspec	2014-08-26 01:04:42.591831800 -0500
+++ src/tilt-1.4.1/tilt.gemspec	2014-08-26 01:05:05.312216900 -0500
@@ -46,7 +46,6 @@ Gem::Specification.new do |s|
       s.add_development_dependency(%q<sass>, [">= 0"])
       s.add_development_dependency(%q<wikicloth>, [">= 0"])
       s.add_development_dependency(%q<yajl-ruby>, [">= 0"])
-      s.add_development_dependency(%q<rdoc>, [">= 0"])
     else
       s.add_dependency(%q<asciidoctor>, [">= 0.1.0"])
       s.add_dependency(%q<RedCloth>, [">= 0"])
